charts/adoption-web/values.yaml

```diff
@@ -29,7 +29,6 @@ nodejs:
     EQUALITY_URL: 'https://pcq.{{ .Values.global.environment }}.platform.hmcts.net'
     DOCUMENT_MANAGEMENT_URL: 'http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal'
     CDAM_API_URL: 'http://ccd-case-document-am-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal'
-    LOCATION_API_URL: 'http://rd-location-ref-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal'
```

Comment:
Please do not remove this url as it is being used. to access the court ref data

Reply:
I remove it because there is a duplicate and it wouldn't let me commit

Reply:
@ShakiraKP @rajatkgupta29 here  by mistake there are two enteries of LOCATION_API_URL. It's a valid change.